docs/topics/about-vscode-extension.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+
+[id='about-vscode-extension_{context}']
+= About the {ProductName} extension for Visual Studio Code
+
+The {ProductName} ({ProductShortName}) extension provides assistance directly in Visual Studio Code for developers migrating or modernizing projects that contain Java applications.
+
+The {ProductShortName} extension analyzes your projects using customizable rulesets, marks issues in the source code, provides guidance to fix the issues, and offers automatic code replacement, if possible.
+
+[IMPORTANT]
+====
+The {ProductName} ({ProductShortName}) extension for Visual Studio Code is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+See link:{KBArticleTechnologyPreview}[Technology Preview Features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
```

Comment:
"support scope" should be capitalized because the web site uses title case and that appears to be what you have adopted here. Either is fine, when referring to an external source, but you need to be consistent.